src/main/archetype/pom.xml

```diff
@@ -76,8 +76,8 @@
 #end
         <core.wcm.components.version>2.19.0</core.wcm.components.version>
 #if ( $includeCommerce == "y" )
-        <core.cif.components.version>2.10.0</core.cif.components.version>
-        <magento.graphql.version>9.0.0-magento242ee</magento.graphql.version>
+        <core.cif.components.version>2.11.0</core.cif.components.version>
```

Comment:
You should probably update the sdk version as well as this is not a hard dependency.

Also I am not sure if the graphql.client dependency is event required anymore (considering the SDK is required for cloud and onprem now)

Reply:
The SDK version was already updated here https://github.com/adobe/aem-project-archetype/commit/a61384f76eaeed7b41a98264f61d0c86e84f7061#diff-8d2547db69a77df225119beca89eeb18180b73f993ae465fc00d5e5b32985486R85 .

The graphql client can be removed indeed and dependencies can be simplified.
I've updated the PR.

Reply:
what about the all package? The graphql-client is still embedded there. This should not be necessary anymore either.